pandas/plotting/_core.py

```diff
@@ -961,7 +961,7 @@ def __call__(self, *args, **kwargs):
             return plot_backend.plot(self._parent, x=x, y=y, kind=kind, **kwargs)
 
         if kind not in self._all_kinds:
-            raise ValueError(f"{kind} is not a valid plot kind")
+            raise ValueError(f"{kind} is not a valid plot kind\nValid plot kinds {self._all_kinds}")
```

Comment:
Can you just add a new line in the file instead of `\n`?

Reply:
```
raise ValueError(f"""{kind} is not a valid plot kind
Valid plot kinds {self._all_kinds}""")
```
Like this? Otherwise it will raise value error and will not go on next line. Maybe I can completely remove \n

Reply:
You can check out examples in the repo where there are multi-line exception messages.

Reply:
Done